src/settings-renderer/components/menu-preview/MenuPreview.tsx

```diff
@@ -438,7 +439,7 @@ export default function MenuPreview() {
           // Only select the child if it is not currently selected. Also, during touch
           // interaction, onPointerUp is also called at the end of a drag operation. In
           // this case we do not want to select the child.
-          else if (!selected && dragIndex === null) {
+          else if (!selected && dragIndex === null && clickedDownContainer === currentContainer.current) {
```

Comment:
You can run `npm run format` to fix such issues.


```suggestion
          else if (
            !selected &&
            dragIndex === null &&
            clickedDownContainer === currentContainer.current
          ) {
```